server/src/entities/test.ts

```diff
@@ -0,0 +1,7 @@
+export type TestResults = {
+    patientId: number;
+    testResult: string;
```

Comment:
Would it be more clear if this field had a type of an enum `TestResultType` which had the values of `POSITIVE` and `NEGATIVE`, you can check the `Role` or `Gender` enums for examples

```suggestion
    result: TestResultType;
```